CONTRIBUTING.md

```diff
@@ -1,126 +1,68 @@
-- `journal` contains the append-only log used by the consensus algorithm
-- `snapshots` abstracts how state snapshots (i.e. `zb-db`) are handled
-
-### Test Execution
-
-Tests can be executed via maven (`mvn verify`) or in your preferred IDE. The Zeebe Team uses mostly [Intellij IDEA](https://www.jetbrains.com/idea/), where we also [provide settings for](https://github.com/camunda/camunda/tree/main/.idea).
-
-> [!TIP]
-> To execute the tests quickly, run `mvn verify -Dquickly -DskipTests=false`.
-> The tests will be skipped when using `-Dquickly` without `-DskipTests=false`.
-
-#### Test Troubleshooting
-
-- If you encounter issues (like `java.lang.UnsatisfiedLinkError: failed to load the required native library`) while running the test StandaloneGatewaySecurityTest.shouldStartWithTlsEnabled take a look at https://github.com/camunda/camunda/issues/10488 to resolve it.
+# Contributing to Camunda
+
+We welcome new contributions. We take pride in maintaining and encouraging a friendly, welcoming, and collaborative community.
+
+Anyone is welcome to contribute to Camunda! The best way to get started is to choose an existing [issue](#starting-on-an-issue).
+
+Core code contributions must fit into the scope of Camunda products. The ultimate decision is up to the core Camunda team. If your vision does not fit in the core Camunda product, you may be directed to the [Camunda Community Hub](https://github.com/camunda-community-hub) and/or [Camunda Marketplace](https://marketplace.camunda.com/en-US/home) instead.
+
+- [Prerequisites](#prerequisites)
+  - [Contributor License Agreement](#contributor-license-agreement)
+  - [Code of Conduct](#code-of-conduct)
+- [GitHub issue guidelines](#github-issue-guidelines)
+  - [Starting on an issue](#starting-on-an-issue)
+- [Creating a pull request](#creating-a-pull-request)
+- [Reviewing a pull request](#reviewing-a-pull-request)
+  - [Review emoji code](#review-emoji-code)
+  - [Stale pull requests](#stale-pull-requests)
+- [Backporting changes](#backporting-changes)
+- [Commit message guidelines](#commit-message-guidelines)
+  - [Commit message header](#commit-message-header)
+  - [Commit message body](#commit-message-body)
+- [Build Camunda from source](#build-camunda-from-source)
+  - [Test Execution](#test-execution)
+    - [Test Troubleshooting](#test-troubleshooting)
+  - [Build profiling](#build-profiling)
+
+## Prerequisites
+### Contributor License Agreement
+
+You will be asked to sign our [Contributor License Agreement](https://cla-assistant.io/camunda-community-hub/community) when you open a Pull Request. We are not
+asking you to assign copyright to us but to give us the right to distribute
+your code without restriction. We ask this of all contributors to
+assure our users of the origin and continuing existence of the code. 
 
-### Build profiling
+*Note: In most cases, you will only need to sign the CLA once.* 
```

Comment:
🔧 Pleae use one of [GitHub's markdown alerts](https://github.com/orgs/community/discussions/16925) for notes

```md
> [!NOTE]  
> In most cases, you will only need to sign the CLA once.
```

> [!NOTE]  
> In most cases, you will only need to sign the CLA once.